### ISSUE TITLE
Module '"vue"' has no exported member 'mergeDefaults'

### ISSUE THREAD

**Issue by sunday60004** [qmhc/vite-plugin-dts#188]
### Describe the bug

我在vue3中引入外部文件中的props对象，并赋值给withDefaults的第二个参数，然后使用dts生成声明文件时，会出现”Module '"vue"' has no exported member 'mergeDefaults'“报错，但实际上代码本身可以正常运行，请问我该怎么处理避免这个报错出现呢？
![image](https://user-images.githubusercontent.com/28419917/224948072-3b523a74-d7a8-4b7d-9c0c-84b1a5133df1.png)

![image](https://user-images.githubusercontent.com/28419917/224947783-f9704ff3-edcb-4cbf-8b0c-51c44ca0f819.png)

![image](https://user-images.githubusercontent.com/28419917/224947573-475e1dcf-747e-44fb-af26-8296b70243cd.png)

### Reproduction

http://www.google.com

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (20) x64 12th Gen Intel(R) Core(TM) i9-12900H       
    Memory: 6.29 GB / 15.70 GB
  Binaries:
    Node: 14.18.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - C:\Program Files\nodejs\yarn.CMD
    npm: 6.14.15 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (110.0.1587.69)
    Internet Explorer: 11.0.19041.1566
  npmPackages:
    @vitejs/plugin-vue: ^4.0.0 => 4.0.0 
    vite: ^4.1.4 => 4.1.4 
    vite-plugin-dts: ^2.1.0 => 2.1.0
```

### Validations

- [X] Read the [FAQ](https://github.com/qmhc/vite-plugin-dts#faq).
- [X] Check that there isn't [already an issue](https://github.com/qmhc/vite-plugin-dts/issues) that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by axtlive** [comment]
I had the same problem

**Comment by webdevnerdstuff** [comment]
I'm also running into this error when doing my build.

Here is one example where it's happening for me: 
https://github.com/webdevnerdstuff/vuetify-inline-fields/blob/main/src/plugin/VInlineCheckbox.vue#L117

To replicate the issue if you clone the repo and run `pnpm i` then `pnpm build`.

Error output example:
```src/plugin/VInlineCheckbox.vue.ts:2:10 - error TS2305: Module '"vue"' has no exported member 'mergeDefaults'.

2 import { mergeDefaults as _mergeDefaults, defineComponent as _defineComponent } from 'vue'
           ~~~~~~~~~~~~~
src/plugin/VInlineCheckbox.vue.ts:8:2 - error TS6133: 'VInlineCheckboxProps' is declared but its value is never read.

8  VInlineCheckboxProps,```

**Comment by veco9** [comment]
I'm currently facing a similar issue as I'm trying to wrap and expand an existing component from a third-party library within a new component. My approach involves combining the default properties provided by the library with the ones I have defined. Here's an example:

```
const props = withDefaults(
  defineProps<
    IThirdPartyLibraryProps & { myNewProperty1: string; myNewProperty2: string }
  >(),
  {...thirdPartyLibraryDefaults, ...myNewDefaults}
);
```
with myNewDefaults:
`const myNewDefaults = {myNewProperty1: "foo", myNewProperty2: "bar"};`

I've found a comment in vue@3.3.4 "runtime-core.d.ts":
`/* removed internal: mergeDefaults */`

I guess this plugin should get an update?
